app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,80 @@
+<nav id="navcontainer" class="navbar navbar-default navbar-fixed-top">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <div id="date" class="navbar-brand">
+        <% if @count -%>
+        <span id="badge_count" class="number_badge"><%= @count %></span>
```

Comment:
After #1924, port the badge styles out of legacy and into `.bootstrap`. That will help us keep straight what's actually needed under the new design.